CMakeLists.txt

```diff
@@ -84,6 +86,20 @@ if( ALGEBRA_PLUGINS_SETUP_GOOGLETEST )
    endif()
 endif()
 
+# Set up Google Benchmark
+option( ALGEBRA_PLUGINS_SETUP_GOOGLEBENCHMARK
```

Comment:
Let's call it `BENCHMARK` rather than `GOOGLEBENCHMARK` as we do for detray.

Reply:
I was actually copying VecMem here, where they called it [`GOOGLEBENCHMARK`](https://github.com/acts-project/vecmem/blob/df310ef85a4ec74234f43e18ccbf472c70ba9e8e/benchmarks/CMakeLists.txt#L9). Nevertheless, I have changed it to be more in line with the `algebra-plugins` way of doing things.